### PR TITLE
Feature/general purpose entity

### PR DESCRIPTION
**Pull request by heikomat**
## What did you change?

I added a general-purpose-entity (`StoreEntity`) to the skeleton-servers (the regular one and the demo-one). the pi- and the browser-skeleton are still missing this entity, see #31.

## How can others test the changes?

With that change, you'll be able to use the new StoreEntity to save arbitrary data to the DB

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: process-engine-server/bin/start.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ const iocModuleNames = [
   '@essential-projects/routing',
   '@essential-projects/timing',
   '@essential-projects/validation',
+  '..'
```
Comment (alexanderkasten) [inline process-engine-server/bin/start.js]: what is this for ?

Comment (heikomat) [reply process-engine-server/bin/start.js]: it makes the start-script add the ioc_module in the parent-folder to the list of ioc_modules to register. This registers the general-purpose entity to the process-engine

Comment (cmg-dev) [reply process-engine-server/bin/start.js]: @heikomat we would add a `,` to the end of the line, not?

Comment (heikomat) [reply process-engine-server/bin/start.js]: @cmg-dev you are correct

**Pull request merged**